.github/scripts/prepare-vr-screenshots-for-upload.js

```diff
@@ -0,0 +1,46 @@
+// @ts-check
+
+const { join } = require('node:path');
+const { existsSync, cpSync, mkdirSync, writeFileSync } = require('node:fs');
+const { createProjectGraphAsync } = require('@nx/devkit');
+
+module.exports = main;
+
+/**
+ *
+ * @param {import('../../scripts/triage-bot/src/types.ts').GithubScriptsParams & {config:{projects:string[]}} } options
+ * @returns
+ */
+async function main(options) {
```

Comment:
- aggregates screenshots generated during `nx affected` under `/screenshots` directory
- generates `screenshots-report.json` with metadata for further processing within `*-comment` workflow ( wish this functionality would be baked in within the vrt tool like we have for bundle size )